deploy/dev/test.sh

```diff
@@ -20,5 +20,11 @@ done
 export PYTHONPATH=$(pwd)
 
 PGPASSWORD="$POSTGRES_PASSWORD" psql -h "$POSTGRES_HOST" -U "$POSTGRES_USER" -d template1 -c 'CREATE EXTENSION IF NOT EXISTS citext;'
-coverage run --source apps -m pytest "$@" && echo && coverage report
+coverage run --source apps -m pytest "$@"
+pytest_exit=$?
+
+echo
+(( pytest_exit == 0 )) && coverage report
```

Comment:
To me that looked like an unintended linebreak at first...

I'd suggest this:

```suggestion
echo ""
(( pytest_exit == 0 )) && coverage report
```